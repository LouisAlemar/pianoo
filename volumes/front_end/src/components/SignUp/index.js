import React, { Component } from 'react';

class SignUp extends Component{
	render(){
		return(
			<div>
				<h2>Sign-Up</h2>
				<form action="">
					<input type="text"/>
					<input type="text"/>
					<button>Submit</button>
				</form>
			</div>
		)
	}
}

export default SignUp;