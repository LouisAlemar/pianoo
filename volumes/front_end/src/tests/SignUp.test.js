import React from 'react';
import renderer from 'react-test-renderer';
import SignUp from '../components/SignUp';

describe("SignUp component", () => {
	test("Matches the snapshot", () => {
		const tree = renderer.create(<SignUp />).toJSON();
		expect(tree).toMatchSnapshot();
	})
})