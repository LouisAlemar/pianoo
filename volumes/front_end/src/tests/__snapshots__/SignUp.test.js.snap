// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp component Matches the snapshot 1`] = `
<div>
  <h2>
    Sign-Up
  </h2>
  <form
    action=""
  >
    <input
      type="text"
    />
    <input
      type="text"
    />
    <button>
      Submit
    </button>
  </form>
</div>
`;
